core/msd/src/main/rdl/TransportPolicyRule.tdl

```diff
@@ -15,12 +15,26 @@ type TransportPolicyProtocol Enum {
     UDP
 }
 
+// Types of transport policy traffic direction
+type TransportPolicyTrafficDirection Enum {
```

Comment:
Who is using this object?

Reply:
Thanks for catching this, this will be part of the request object TransportPolicyValidateRule
will update the PR